vuln/npm/400.json

```diff
@@ -0,0 +1,22 @@
+{
+  "id": 400,
+  "title": "Denial of Service",
+  "overview": "`https-proxy-agent` passes unsanitized options to Buffer(arg), resulting in DoS and uninitialized memory leak",
+  "created_at": "2018-02-25",
+  "updated_at": "2018-04-02",
+  "publish_date": "2018-04-02",
+  "author": "Сковорода Никита Андреевич (https://github.com/ChALkeR)",
+  "module_name": "https-proxy-agent",
+  "cves": [
+    ""
```

Comment:
This fails lint. Please use an empty array instead: `[]`.
Also, previous entries where this was missed need fixing.

Reply:
thanks, good catch